_includes/live-demos/premiumskinsandicons-fluent/index.html

```diff
@@ -0,0 +1,6 @@
+<div class="dummy-header">
```

Comment:
Should that class be going into prod?

Reply:
I'll leave that Q to @lostkeys

Reply:
Yes its a dummy app bar, the same one we use on fabric which this is an evolution of.